infoportal-client/src/features/Protection/DashboardMonito/ProtectionDashboardMonitoSample.tsx

```diff
@@ -125,16 +125,40 @@ export const ProtectionDashboardMonitoSample = ({
               {_ => agDisplay === 'chart' ? (
                 <ChartBarStacker data={_} height={250}/>
               ) : (
-                <Sheet
+                <Datatable
                   id="prot-dash-population"
                   sx={{border: t => `1px solid ${t.palette.divider}`, overflow: 'hidden', borderRadius: t => t.shape.borderRadius + 'px'}}
                   hidePagination
                   data={_}
                   columns={[
-                    {width: 0, id: 'Group', head: m.ageGroup, type: 'select_one', render: _ => _.key},
-                    {width: 0, id: 'Male', head: m.male, type: 'number', renderValue: _ => _.Male, render: _ => formatLargeNumber(_.Male)},
-                    {width: 0, id: 'Female', head: m.female, type: 'number', renderValue: _ => _.Female, render: _ => formatLargeNumber(_.Female)},
-                    {width: 0, id: 'Other', head: m.other, type: 'number', renderValue: _ => _.Other ?? 0, render: _ => formatLargeNumber(_.Other ?? 0)},
+                    {
+                      width: 0,
+                      id: 'Group',
+                      head: m.ageGroup,
+                      type: 'select_one',
+                      renderQuick: _ => _.key
+                    },
+                    {
+                      width: 0,
+                      id: 'Male',
+                      head: m.male,
+                      type: 'string',
+                      renderQuick: _ => formatLargeNumber(_.Male)
```

Comment:
As for 'date' it's important to return a `value` that can be used to sort and display charts. Her `type` should be `number`. So we need to return a label with the format function and value as simple number.